ubuntu/gpu/python/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM sarthfrey/gpu-cuda:10.1
```

Comment:
we'll have to fix this before merging. When you're ready to build the images I can push them to databricksruntime public docker hub and you can update these.